style.css

```
@@ -0,0 +1,68 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Arvo:wght@700&family=Cutive+Mono&family=Montserrat:ital,wght@0,100;1,900&family=Recursive:wght@300&family=Roboto+Mono:wght@100&family=Splash&display=swap');
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: 'Arvo', serif;
+}
+body{
+    min-height: 100vh;
+    background-color: #bababa;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.paper{
+    position: relative;
+    width: 90%;
+    min-width: 400px;
+    max-width: 800px;
+    height: 480px;
+    margin: 0 auto;
+    background-color: #fafafa;
+    border-radius: 10px;
+    overflow: hidden;
+}
+
+.paper::before{
+    content: '';
+    position: absolute;
+    top: 0;
+    left:0;
+    bottom: 0;
+    width:60px;
+    border-right:4px solid #d18383;
+    background: radial-gradient(rgb(88, 85, 85) 6px, transparent 7px) repeat-Y;
+background-size: 30px 30px;
+box-sizing: border-box;
+}
+::-webkit-scrollbar{
+    display: none;
+}
+
+.paper-content{
+    position: absolute;
+    top: 30px;
+    right:0;
+    bottom: 30px;
+    left:60px;
+    background: linear-gradient(transparent ,transparent 28px,skyblue 28px);
```

```suggestion
    background: linear-gradient(transparent 28px,skyblue 28px);
```

I think you can remove the first transparent, the code won't get affected

---

> I think you can remove the first transparent, the code won't get affected

yes, let me update it